### DISCUSSION TITLE
About parameter --genomeDir

### DISCUSSION THREAD

**Discussion by EdJarvis0** [alexdobin/STAR#1467] [Q&A]
Here are my reference files in GTF folder, and I don"t know what's wrong.It keep logging:"EXITING because of FATAL ERROR: could not open genome file /home/liuzijia/桌面/Drop-Seq/GTF/Mus_musculus.GRCm39.105.chr.gtf//genomeParameters.txt"
Does it mean that I lack of "genomeParameters.txt"? But I get my GTF file from the ENSEMBL, I can't understand.
Thanks.

**Comment by alexdobin** [comment]
Hi @EdJarvis0 

I think there is an issue with the command line formatting.
Please send me the Log.out file.

Cheers
Alex

**Reply by EdJarvis0** [reply]
I think i have solved this question. Anyway thanks a lot~

Best
Jia

**Comment by EdJarvis0** [comment]
Just follow the manual to creat relative files, then run the STAR.
